CHANGELOG.md

```diff
@@ -83,6 +87,34 @@ Changelogs for versions not listed here can be found at https://github.com/DataD
   - pymongo: Ensures instances of the `pymongo.MongoClient` can be patch after pymongo is imported
 
 
+---
+
+## 2.13.1
+
+
+### New Features
```

Comment:
nit: Could we alphabetize the sub-categories within the change types (e.g. within "Bug Fixes" change the ordering to Code Security > Profiling > Tracing)